File: src/main/java/net/nodus/NodusApplication.java
package net.nodus;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableFeignClients
@EnableScheduling
@EnableJpaAuditing
public class NodusApplication {

    public static void main(String[] args) {
        SpringApplication.run(NodusApplication.class, args);
    }
}

File: src/main/kotlin/net/nodus/NodusApplication.kt
package net.nodus

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.data.mongodb.config.EnableMongoAuditing
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableFeignClients
@EnableScheduling
@EnableMongoAuditing
class NodusApplication

fun main(args: Array<String>) {
	runApplication<NodusApplication>(*args)
}
